projects/angular/src/layout/tabs/_tabs.clarity.scss

```
@@ -92,54 +81,38 @@ button.nav-link {
     box-shadow: none;
     flex-direction: column;
     align-items: stretch;
-    margin-right: variables.$clr_baselineRem_1;
+    margin-right: #{tokens.$cds-global-space-9};
     overflow: auto;
     flex-shrink: 0;
-    padding: variables.$clr_baselineRem_4px;
-    width: variables.$clr_baselineRem_10;
-    min-width: variables.$clr_baselineRem_2;
+    padding: #{tokens.$cds-global-space-3};
+    width: calc(10 * #{tokens.$cds-global-space-9});
```

Use `baselinePx` function for calculation to be more obvious, but is fine as it is, too.

```suggestion
    width: mixins.baselinePx(240);
```